content/en/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,33 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title:    Security For Windows Nodes
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+This page describes security considerations and best practices specific to the Windows operating system.
+
+<!-- body -->
+
+On Windows, data from Secrets are written out in clear text onto the node's local
+storage (as compared to using tmpfs / in-memory filesystems on Linux). As a cluster
+operator, you should take both of the following additional measures:
+
+1. Use file ACLs to secure the Secrets' file location.
+1. Apply volume-level encryption using [BitLocker](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-how-to-deploy-on-windows-server).
+
+[RunAsUsername](/docs/tasks/configure-pod-container/configure-runasusername)
+can be specified for Windows Pods or containers to execute the container
+processes as a node-default user. This is roughly equivalent to
+[RunAsUser](/docs/concepts/policy/pod-security-policy/#users-and-groups).
+
+Linux-specific pod security context privileges such as SELinux, AppArmor, Seccomp, or capabilities (POSIX capabilities), and others are not supported.
+
+Privileged containers are [not supported](#compatibility-v1-pod-spec-containers-securitycontext) on Windows.
+Instead [HostProcess containers](/docs/tasks/configure-pod-container/create-hostprocess-pod) can be used on Windows to perform many of the tasks performed by privileged containers on Linux.
```

Comment:
I think this belongs elsewhere, perhaps as a note in https://kubernetes.io/docs/concepts/workloads/pods/#privileged-mode-for-containers ?

Maybe at least restate it there.

Reply:
Hmm, this is already stated at https://kubernetes.io/docs/concepts/workloads/pods/#privileged-mode-for-containers

> If your cluster has the WindowsHostProcessContainers feature enabled, you can create a [Windows HostProcess pod](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the windowsOptions.hostProcess flag on the security context of the pod spec. All containers in these pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used to perform administrative tasks as is done with Linux privileged containers.

Should we just remove mention of it here?

Reply:
It's a little bit tidier to use the same or very similar wording in both places; it's also fine to leave them different and then after this merges log a good-first-issue about making the two places match.
(We like to have a nice pool of easily tractable fixes).

Reply:
I think this would qualify as an easily tractable fix so I can open that issue and add the help-wanted tag after this merges if that could help new contributors get started making PRs into this repo.